lib/versioning/hex/index.spec.ts

```diff
@@ -198,7 +198,7 @@ describe(getName(__filename), () => {
         currentVersion: '1.2.3',
         newVersion: '2.0.7',
       })
-    ).toEqual('>= 1.0.0 or <= 2.0.7');
+    ).toEqual('>= 1.0.0 or <= 2.0.0');
```

Comment:
Need to rewrite this tests later maybe, it seems like copy-paste from `handles and`